src/modules/ChannelSettings/components/ModerationPanel/OperatorList.tsx

```diff
@@ -5,63 +5,38 @@ import React, {
   useCallback,
   useContext,
 } from 'react';
+import type { User } from '@sendbird/chat';
 
 import { LocalizationContext } from '../../../../lib/LocalizationContext';
+import { useChannelSettingsContext } from '../../context/ChannelSettingsProvider';
+
 import Button, { ButtonTypes, ButtonSizes } from '../../../../ui/Button';
-import IconButton from '../../../../ui/IconButton';
-import Icon, { IconTypes, IconColors } from '../../../../ui/Icon';
-import ContextMenu, { MenuItem, MenuItems } from '../../../../ui/ContextMenu';
+import UserListItemMenu from '../../../../ui/UserListItemMenu/UserListItemMenu';
 
 import UserListItem from '../UserListItem';
 import OperatorsModal from './OperatorsModal';
 import AddOperatorsModal from './AddOperatorsModal';
-import useSendbirdStateContext from '../../../../hooks/useSendbirdStateContext';
-import { useChannelSettingsContext } from '../../context/ChannelSettingsProvider';
-import { User } from '@sendbird/chat';
 
 export const OperatorList = (): ReactElement => {
   const [operators, setOperators] = useState<User[]>([]);
   const [showMore, setShowMore] = useState(false);
   const [showAdd, setShowAdd] = useState(false);
   const [hasNext, setHasNext] = useState(false);
   const { stringSet } = useContext(LocalizationContext);
-
-  const state = useSendbirdStateContext();
   const { channel } = useChannelSettingsContext();
 
-  const userId = state?.config?.userId;
-
-  useEffect(() => {
+  const refreshList = useCallback(() => {
     if (!channel) {
       setOperators([]);
       return;
     }
-
-    const operatorListQuery = channel?.createOperatorListQuery({
-      limit: 10,
-    });
+    const operatorListQuery = channel?.createOperatorListQuery({ limit: 10 });
     operatorListQuery.next().then((operators) => {
       setOperators(operators);
       setHasNext(operatorListQuery.hasNext);
     });
   }, [channel]);
-
-  const refreshList = useCallback(
-    () => {
-      if (!channel) {
-        setOperators([]);
-        return;
-      }
-      const operatorListQuery = channel?.createOperatorListQuery({
-        limit: 10,
-      });
-      operatorListQuery.next().then((operators) => {
-        setOperators(operators);
-        setHasNext(operatorListQuery.hasNext);
-      });
-    },
-    [channel],
-  );
+  useEffect(refreshList, [channel]);
```

Comment:
```suggestion
  useEffect(refreshList, [channel?.url]);
```
Let's put a simplest data in the deps array. Same for line number 38.